velox/docs/configs.rst

```diff
@@ -1113,3 +1113,45 @@ Tracing
      - false
      - If true, we only collect the input trace for a given operator but without the actual
        execution. This is used for crash debugging.
+
+Cudf-specific Configuration (Experimental)
+--------
+These configurations are available when compiled with cuDF.
+Note: These configurations are experimental and subject to change.
+
+.. list-table::
+   :widths: 30 10 10 70
+   :header-rows: 1
+
+   * - Property Name
+     - Type
+     - Default Value
+     - Description
+   * - cudf.enabled
+     - bool
+     - true
+     - If true, enable cuDF. By default, it is enabled if compiled with cuDF.
+   * - cudf.memory_resource
+     - string
+     - async
+     - The memory resource to use for cuDF. Possible values are (cuda, pool, async, arena, managed, managed_pool).
+   * - cudf.memory_percent
+     - integer
+     - 50
+     - The initial percent of GPU memory to allocate for pool or arena memory resources.
+   * - cudf.function_name_prefix
+     - string
+     - ""
+     - The prefix to use for the function names in cuDF.
+   * - cudf.force_replace
+     - bool
+     - false
+     - If true, force replacement of CPU operators with cuDF operators.
+   * - cudf.debug_enabled
+     - bool
+     - false
+     - If true, enable debug printing.
```

Comment:
Any particular reason we are not using stand way to configure GLOG?

Reply:
This is addressed in PR https://github.com/facebookincubator/velox/pull/15321